repository/MonticelloFileTree-Core.package/MCFileTreeRepository.class/instance/readerClassForFileNamed..st

```diff
@@ -0,0 +1,4 @@
+actions
+readerClassForFileNamed: fileName
+	^ "MCReader readerClassForFileNamed: fileName"
+		MCFileTreeReader
```

Comment:
Actually `MCReader` isn't quite right either ... need to be able to preserve the old behavior of MCFileTreeRepository, while splicing in the Cypress reader/writer